cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+| repodata.json.bz2    | `repodata.json.bz2` file    | application/vnd.conda.repodata.v1+json+bz2     |
+| channeldata.json     | `channeldata.json` file     | application/vnd.conda.channeldata.v1+json      |
+| run_exports.json     | `run_exports.json` file     | application/vnd.conda.run_exports.v1+json      |
+| run_exports.json.zst | `run_exports.json.zst` file | application/vnd.conda.run_exports.v1+json+zst  |
+| run_exports.json.gz  | `run_exports.json.gz` file  | application/vnd.conda.run_exports.v1+json+gzip |
+| run_exports.json.bz2 | `run_exports.json.bz2` file | application/vnd.conda.run_exports.v1+json+bz2  |
+| repodata_from_packages.json | `repodata_from_packages.json` file | application/vnd.conda.repodata_from_packages.v1+json |
+| repodata_from_packages.json.zst | `repodata_from_packages.json.zst` file | application/vnd.conda.repodata_from_packages.v1+json+zst |
+| repodata_from_packages.json.gz | `repodata_from_packages.json.gz` file | application/vnd.conda.repodata_from_packages.v1+json+gzip |
+| repodata_from_packages.json.bz2 | `repodata_from_packages.json.bz2` file | application/vnd.conda.repodata_from_packages.v1+json+bz2 |
+| patch_instructions.json | `patch_instructions.json` file | application/vnd.conda.patch_instructions.v1+json |
+| patch_instructions.json.zst | `patch_instructions.json.zst` file | application/vnd.conda.patch_instructions.v1+json+zst |
+| patch_instructions.json.gz | `patch_instructions.json.gz` file | application/vnd.conda.patch_instructions.v1+json+gzip |
+| patch_instructions.json.bz2 | `patch_instructions.json.bz2` file | application/vnd.conda.patch_instructions.v1+json+bz2 |
+
+### OCI Artifact Structure for Conda Packages
+
+In order to store a conda package as an OCI artifact, we need to define how the data within the conda package is mapped
+into the OCI blobs and what additional metadata (if any) is stored in the OCI manifest.
+
+A conda package as an OCI artifact MUST have the following layers:
+
+- a layer holding either the `.tar.bz2` or `.conda` package file with the mediaType
+  `application/vnd.conda.package.v1` or `application/vnd.conda.package.v2`, respectively.
+- a layer holding the `info` folder as a gzipped tarball with extension `.tar.gz` and mediaType `application/vnd.conda.info.v1.tar+gzip`.
+- a layer holding the `info/index.json` file as plain JSON with the mediaType `application/vnd.conda.info.index.v1+json`.
+
+Additional layers are NOT allowed in the OCI artifact. If a conda package exists in both `.tar.bz2` and `.conda` format, the `.conda`
+version MUST be chosen to be in the OCI artifact.
+
+The manifest MUST have the following [OCI Annotations][annotations]
+
+- `org.conda.oci.schema` with the value `1` indicating the `v1` specification (defined by this document).
+- `org.conda.package.name` with the name of the conda package.
+- `org.conda.package.version` with the version of the conda package.
+- `org.conda.package.build` with the build string of the conda package
```

Comment:
Shall we add channel, label and subdir too?

Reply:
subdir yes, but not channel or label.

Reply:
We never hash the subdir so no need actually.